docs/targets/substrate.rst

```diff
@@ -49,3 +49,19 @@ import object.
 
     The import file ``substrate`` is only available when compiling for the Substrate
     target.
+
+Call Flags
+__________
+
+The Substrate contracts pallet knows several 
+`flags <https://github.com/paritytech/substrate/blob/6e0059a416a5768e58765a49b33c21920c0b0eb9/frame/contracts/src/wasm/runtime.rs#L392>`_ 
+that can be used when calling other contracts.
+
+Solang allows a named ``flags`` argument of type ``uint32`` to the ``address.call()`` function to set desired flags.
```

Comment:
```suggestion
Solang allows a ``flags`` call argument of type ``uint32`` to the ``address.call()`` function to set desired flags.
```
We've been calling this syntax `call argument` in Solana.

Reply:
@LucasSte weird I added them all to the suggestion batch. Maybe I misclicked in the so that's why only one of your suggestions went in